src/azure-cli/azure/cli/command_modules/resource/_validators.py

```diff
@@ -16,7 +16,9 @@
 MSI_LOCAL_ID = '[system]'
 
 
-def _validate_template_input(namespace):
+def process_ts_create_or_update_namespace(namespace):
+    from azure.cli.core.commands.validators import validate_tags
+    validate_tags(namespace)
```

Comment:
```
tags_type = CLIArgumentType(
    validator=validate_tags,
    help="space-separated tags: key[=value] [key[=value] ...]. {}".format(quote_text),
    nargs='*'
)
```

`tags_type` already has the logic to call `validate_tags`, why do you call it repeatedly here?

Reply:
To set tags in the namespace to the tags dictionary.
Ref: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/_validators.py for similar usage.